diesel/src/query_dsl/mod.rs

```diff
@@ -332,6 +332,7 @@ pub trait QueryDsl: Sized {
     where
         Self: methods::SelectDsl<CountStar>,
     {
+        #[allow(deprecated)]
```

Comment:
If we choose that solution we need to think about if we really want to put a `#[deprecated]` onto the old type aliases as this results in warnings as soon as you have an import like that, even if that import could refer to the function as well.